src/nvim/os/win_defs.h

```diff
@@ -6,16 +6,22 @@
 #define TEMP_DIR_NAMES {"$TMP", "$TEMP", "$USERPROFILE", ""}
 #define TEMP_FILE_PATH_MAXLEN _MAX_PATH
 
-// Defines needed to fix the build on Windows:
-// - DFLT_DIR
-// - DFLT_BDIR
-// - DFLT_VDIR
-// - EXRC_FILE
-// - VIMRC_FILE
-// - SYNTAX_FNAME
-// - DFLT_HELPFILE
-// - SYS_VIMRC_FILE
-// - SPECIAL_WILDCHAR
+// Windows system-dependent file names
+#ifndef SYS_VIMRC_FILE
+# define SYS_VIMRC_FILE "$VIM\\sysinit.vim"
+#endif
+#ifndef DFLT_HELPFILE
+# define DFLT_HELPFILE  "$VIMRUMTIME\\doc\\help.txt"
+#endif
+#ifndef SYNTAX_FNAME
+# define SYNTAX_FNAME   "$VIMRUNTIME\\syntax\\%s.vim"
```

Comment:
:+1:

Why don't we move `PAHTSEPSTR` to `unix_defs.h`/`win_defs.h` as appropriate since
`ascii.h` includes `os_defs.h`.

Ok?